src/main/resources/db/migration/V1_113__add_3oct_contract_types.sql

```diff
@@ -0,0 +1,71 @@
+;
+
+INSERT INTO desired_outcome (id, service_category_id, description)
+-- PRJ_7672 - D&R Northumbria Community only
+VALUES ('79e00aae-c8e5-4d9d-8e49-55d3f64cf84d', '748732c2-7f94-4966-b5f2-d19e86e0cbf2',
+        'Service user achieves abstinence or controlled/ non-dependent or non-problematic substance misuse.')
+     , ('458ec1ed-930c-4af9-9acf-b7303ed84c65', '748732c2-7f94-4966-b5f2-d19e86e0cbf2',
+        'Service user improves their physical health and mental resilience.')
+     , ('a1e43eeb-72ad-413b-af3e-03383d7bfe1a', '748732c2-7f94-4966-b5f2-d19e86e0cbf2',
+        'Service user enhances their skills to manage risky situations which may pose a trigger or relapse.')
+     , ('9ac26331-b390-486e-be2b-3e9a07c366e1', '748732c2-7f94-4966-b5f2-d19e86e0cbf2',
+        'Service user enhances their belief in ability to manage/ desist from addiction(s).')
+     , ('ab96749c-8c15-4b5c-94d7-666432218bf9', '748732c2-7f94-4966-b5f2-d19e86e0cbf2',
+        'Service user increases understanding of addictive behaviours and triggers and options to reduce dependency.')
+     , ('d51618fa-ec50-47de-8438-8c7ba75c5992', '748732c2-7f94-4966-b5f2-d19e86e0cbf2',
+        'Service user establishes Dependency Pathways to manage a range of addictive behaviours, including supporting access into other treatment providers and detox programmes.')
+-- PRJ_7659 - FBD Wales
+    , ('ec9b60e2-7732-4747-94c7-f3904d5fb3d4', 'cf2946a3-6bf2-4f0c-9154-190c2b2f899f',
+        'Service user has access to appropriate financial products, advice and/or services.')
+     , ('ba5c6553-f2c5-4b2b-8a2f-7f6aec266aa2', 'cf2946a3-6bf2-4f0c-9154-190c2b2f899f',
+        'Pathways are established to help service user maintain and sustain an income, safely manage money and reduce debt.')
+     , ('8e40c026-b7b5-4d20-bbc9-891f697a70fa', 'cf2946a3-6bf2-4f0c-9154-190c2b2f899f',
+        'Service user on probation financial management skills are developed and/or enhanced including online banking.')
+     , ('95370c1b-e4db-403d-b595-d22105f9c479', 'cf2946a3-6bf2-4f0c-9154-190c2b2f899f',
+        'Service user can successfully navigate the benefits system.')
+     , ('a670adae-9e36-434d-8c4d-1ebdda3f7b15', 'cf2946a3-6bf2-4f0c-9154-190c2b2f899f',
+        'Service user gains quick access to universal credit.')
+-- PRJ_7647 FBD NE
+     , ('6ba59e39-6c9d-4aaf-bf45-76b55f151a3f', '07e92c9c-51a6-43bb-81a9-226f518dd5ad',
+        'The person referred’s financial management skills are developed and/or enhanced, including online banking skills.')
```

Comment:
_not a blocker_
When we do the simplification, we'll need to change the "fancy" apostrophes `’` for "simple" ones e.g. `'`